### PR TITLE
Cookbook use-case fix and a few minor improvements

### PR DESCRIPTION
**Pull request by ptomato**
Closes: #328

### CODE REVIEW

File: docs/cookbook/absoluteFromLegacyDate.mjs [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
- * Converts a Date instance into a Temporal.Absolute instance
- *
- * @param {Date} esDate This is a Date instance
- * @returns {Temporal.Absolute} Temporal.Absolute instance
- */
-function getAbsoluteFromDate(esDate) {
-  const date = Temporal.Absolute.fromEpochMilliseconds(esDate.getTime());
-  return date;
-}
+const legacyDate = new Date('1970-01-01T00:00:01Z');
```
Comment (gibson042) [inline docs/cookbook/absoluteFromLegacyDate.mjs]: The function-based approach is more general and (perhaps more importantly) testable... why was it abandoned here?

File: docs/cookbook/getParseableZonedStringAtInstant.mjs [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+  const timeZoneObject = Temporal.TimeZone.from(ianaTimeZoneName);
+  return getParseableZonedStringAtInstant(absolute, timeZoneObject);
+}
+
+/**
+ * Converts an absolute point in time to a string with the time in the specified
+ * time zone and a time zone designation.
+ *
+ * @param {Temporal.Absolute} absolute An absolute point in time
+ * @param {string} absolute An absolute point in time
```
Comment (gibson042) [inline docs/cookbook/getParseableZonedStringAtInstant.mjs]: This parameter is misdocumented.
```suggestion
 * @param {Temporal.TimeZone} timeZone The target time zone
```

File: docs/cookbook/absoluteFromLegacyDate.mjs (hunk 1 above)
Comment (Ms2ger) [reply docs/cookbook/absoluteFromLegacyDate.mjs]: Generality isn't really a goal; this is just showing how the API can be used. The code is being tested in CI, so not sure what that's about.

File: docs/cookbook/getParseableZonedStringAtInstant.mjs (hunk 2 above)
Comment (ptomato) [reply docs/cookbook/getParseableZonedStringAtInstant.mjs]: Thanks, opened #489 as a follow up for this.

**Pull request merged**